lib/Xud.ts

```diff
@@ -91,7 +91,8 @@ class Xud extends EventEmitter {
 
       this.pool = new Pool(this.config.p2p, loggers.p2p, this.db.models);
 
-      this.swaps = new Swaps(loggers.swaps, this.pool, this.lndbtcClient, this.lndltcClient);
+      this.swaps = new Swaps(loggers.swaps, this.db.models, this.pool, this.lndbtcClient, this.lndltcClient);
+      await this.swaps.init();
```

Comment:
I think we can add this to the `initPromises` array rather than awaiting it separately.